src/containers/pages/IRS/Map/helpers.ts

```diff
@@ -1,35 +1,89 @@
 import GeojsonExtent from '@mapbox/geojson-extent';
 import { BLACK, GREY, TASK_GREEN, TASK_ORANGE, TASK_RED, TASK_YELLOW } from '../../../../colors';
 import { GisidaProps } from '../../../../components/GisidaWrapper';
-import {
-  circleLayerConfig,
-  fillLayerConfig,
-  IndicatorRowItem,
-  IndicatorRows,
-  lineLayerConfig,
-} from '../../../../configs/settings';
+import { circleLayerConfig, fillLayerConfig, lineLayerConfig } from '../../../../configs/settings';
 import { MAIN_PLAN, STRUCTURE_LAYER } from '../../../../constants';
 import { FlexObject } from '../../../../helpers/utils';
 import { GenericJurisdiction } from '../../../../store/ducks/generic/jurisdictions';
 import { StructureFeatureCollection } from '../../../../store/ducks/generic/structures';
 import { Jurisdiction } from '../../../../store/ducks/jurisdictions';
 
-/** Default indicator stops */
-const defaultIndicatorStops = [
-  ['Complete', TASK_GREEN],
-  ['Not Sprayed', TASK_RED],
-  ['Partially Sprayed', TASK_ORANGE],
-  ['Not Visited', TASK_YELLOW],
-  ['Not Eligible', BLACK],
-];
+export const IRSIndicatorStops: { [key: string]: string[][] } = {
+  namibia2019: [
+    ['Sprayed', TASK_GREEN],
+    ['Not Sprayed', TASK_RED],
+    ['Partially Sprayed', TASK_ORANGE],
+    ['Not Visited', TASK_YELLOW],
+    ['Not Eligible', BLACK],
+  ],
+  zambia2019: [
+    ['Complete', TASK_GREEN],
+    ['Not Sprayed', TASK_RED],
+    ['Partially Sprayed', TASK_ORANGE],
+    ['Not Visited', TASK_YELLOW],
+    ['Not Eligible', BLACK],
+  ],
+};
+
+export interface IndicatorRowItem {
+  denominator: string | number;
+  description: string;
+  numerator: string | number;
+  title: string;
+  value: string | number;
+}
+
+export type IndicatorRows = IndicatorRowItem[];
+
+export const IRSIndicatorRows: { [key: string]: IndicatorRows } = {
```

Comment:
@moshthepitt what do you think of creating a simple getter function for this instead of referencing it like `IRSIndicatorRows[key]`?